src/client/components/LocationPicker/LocationPicker.less

```
@@ -0,0 +1,18 @@
+@import "variables";
```

```suggestion
@import 'variables';
```

I need to add some less linting